regression_tests/stages/clone_demodjango_deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+echo -e "\nGet CodeStar connection details"
+codestar_connections=$(aws codestar-connections list-connections --provider-type GitHub --query "Connections[? ConnectionStatus == 'AVAILABLE']")
+aws_account=$(echo "$codestar_connections" | jq -r ".[0].OwnerAccountId")
+codestar_arn=$(echo "$codestar_connections" | jq -r ".[0].ConnectionArn")
+codestar_connection_id=$(echo "${codestar_arn##*/}")
+
+echo -e "\nClone demodjango_deploy"
```

Comment:
Question, isn't this the generic build image? If so, why is it working directly wtih demodjango-deploy? 🤔

Reply:
The regression tests test the combination of four codebases:

* `demodjango`
* `demodjango-deploy`
* `terraform-platform-modules`
* `platform-tools`

We will be triggering them on merge to main on any of those in due course.